cmip6_downscaling/methods/bcsd/tasks.py

```diff
@@ -171,7 +171,7 @@ def postprocess_bcsd(
         UPath to post-processed dataset.
     """
 
-    target = results_dir / "interpolate_prediction" / run_parameters.run_id
+    target = results_dir / "daily" / run_parameters.run_id
```

Comment:
maybe "post-process" instead of daily?

Reply:
i think that marking it `daily` denotes that it is a final output file whereas `postprocess` doesn't really mean much to me at least? the most important flag seems that it is in the `results_dir`, and from there we distinguish between timesteps. it also plays nice with the monthly/summary files that are named `monthly_summary` and `annual_summary`. but i'd be open to changing the `monthly_summary`